### PR TITLE
Create uicomponentlibrary.md

### PR DESCRIPTION
**Pull request by otoole-brendan**
Draft documentation for UI component library

### CODE REVIEW

File: main/guides/getting-started/uicomponentlibrary.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
+# Welcome to the Agoric UI Component Library
```
Comment (dckc) [inline main/guides/getting-started/uicomponentlibrary.md:1]: any particular reason to put this under `/getting-started/`?

**Comment by dckc** [comment]
I suppose this is w.r.t. #922

File: main/guides/getting-started/uicomponentlibrary.md (hunk 1 above)
Comment (otoole-brendan) [reply main/guides/getting-started/uicomponentlibrary.md:1]: @dckc good catch - I want it to be a separate page on the same hierarchy as 'Getting Started' - not within it. @kbennett2000 might need your help to do that

Comment (kbennett2000) [reply main/guides/getting-started/uicomponentlibrary.md:1]: Sure, we can create a subfolder underneath `main/guides` for all of this

**Comment by otoole-brendan** [comment]
Redundant due to https://github.com/Agoric/documentation/blob/main/main/guides/UIComponentLibrary/UIComponentsstart.md

**Pull request closed without merging**